recipes/mvgavg/LICENSE.txt

```diff
@@ -1,674 +0,0 @@
-                    GNU GENERAL PUBLIC LICENSE
```

Comment:
Other packages in the `recipes` directory shouldn't be modified. The CI is smart enough to ignore them.

Reply:
Got it.

Reply:
You need to restore this file and the `meta.yml` in this directory.

Reply:
I've restored these two files.